config.example.yml

```diff
@@ -24,6 +24,16 @@ server:
 #    Access-Control-Allow-Origin: "*"
```

Comment:
I think we need different header examples here, otherwise It looks like useless settings :D.
```
responseheaders:
  X-Custom-Header: "custom value"
```